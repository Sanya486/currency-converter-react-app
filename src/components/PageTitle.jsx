import React from "react";
import { Title } from "../styled/PageTitle.styled";

// import PropTypes from 'prop-types'

const PageTitle = ({ title }) => <Title>{title}</Title>;

// PageTitle.propTypes = {

// }

export default PageTitle;
