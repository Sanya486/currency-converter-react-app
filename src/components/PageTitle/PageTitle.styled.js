import styled from "styled-components";


export const Title = styled.h1`
text-align: center;
margin-top: 20px;
margin-bottom: 20px;
  font-size: 34px;
  font-weight: 400;
`;
