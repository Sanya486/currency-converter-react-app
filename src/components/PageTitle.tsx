import React, { FC } from "react";
import { Title } from "../styled/PageTitle.styled";

const PageTitle: FC<{ title: string }> = ({ title }) => <Title>{title}</Title>;

export default PageTitle;
