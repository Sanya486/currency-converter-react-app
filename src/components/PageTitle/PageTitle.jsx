import React from 'react'
import {Title} from './PageTitle.styled'

// import PropTypes from 'prop-types'

const PageTitle = ({title}) => {
  return <Title>{title}</Title>;
}

// PageTitle.propTypes = {

// }

export default PageTitle
