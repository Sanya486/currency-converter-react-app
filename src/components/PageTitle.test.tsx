import { render } from "@testing-library/react";
import PageTitle from "./PageTitle";

test("is PageTitle render", () => {
    const { asFragment } = render(<PageTitle title="Hello" />)
    expect(asFragment()).toMatchSnapshot()
})