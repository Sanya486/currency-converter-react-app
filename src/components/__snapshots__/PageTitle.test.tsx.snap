// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is PageTitle render 1`] = `
<DocumentFragment>
  <h1
    class="sc-beyTiQ hRVvUr"
  >
    Hello
  </h1>
</DocumentFragment>
`;
